cli/testdata/coder_server_--help.golden

```diff
@@ -66,6 +66,10 @@ Clients include the coder cli, vs code extension, and the web UI.
       --ssh-hostname-prefix string, $CODER_SSH_HOSTNAME_PREFIX (default: coder.)
           The SSH deployment prefix is used in the Host of the ssh config.
 
+      --web-terminal-renderer string, $CODER_WEB_TERMINAL_RENDERER (default: webgl)
+          The framework to use when rendering the terminal. Valid values are
+          'canvas', 'webgl', or 'dom'.
+
```

Comment:
Is the plan to make canvas the default in another PR if the web terminal renderer is, in fact, the problem? I think in our case it's best to sacrifice performance in favor of reliability.

Unless we know which _specific niche cases_ lead to the rendering bug which excludes 95% of users.

Reply:
i agree - we should default to `canvas` until the upstream Chromium version (119) is released that contains the fix.

Reply:
I know that according to @BrunoQuaresma there are issues with both `canvas` and `webgl`. I made `webgl` the default since it's the default now but if there's an option that has the best experience I would prefer making that the default.